deploy/olm-catalog/performance-addon-operator/4.7.4/performance-addon-operator.v4.7.4.clusterserviceversion.yaml

```diff
@@ -161,8 +172,6 @@ spec:
           strategy: {}
           template:
             metadata:
-              annotations:
-                target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
```

Comment:
we probably don't want to lose this

Reply:
it 4.7, and the feature is available only for 4.8, so it should be ok. I think it was added in first place by mistake :)

Reply:
fair enough